README.md

```diff
@@ -73,6 +73,48 @@ TextRazor.phrases('api_key', 'text')
 
 ```
 
+### Dictionaries
+
+You can manage dictionaries and their entries through the dictionary API.
+
+#### Creating a dictionary
+
+```
+client = TextRazor::Client.new('api_key')
+dictionary = TextRazor::Dictionary.new(id: 'my-dictionary')
+
+client.create_dictionary(dictionary)
+```
+
+#### Adding entries to a dictionary
+
+```
+entry = TextRazor::DictionaryEntry(id: 'my-entry', text: 'Text to be matched')
```

Comment:
`DictionaryEntry` and `Dictionary` should be internal classes, IMHO. The API interface should be something like this:

```
client = TextRazor::Client.new('api_key')
client.create_dictionary('my-dictionary') 
# This can have keyword arguments but I'd not do it if it's not anywhere else in the library, makes it inconsistent
client.create_dictionary_entries('my-dictionary', {id: 'my-entry', text: 'Text to be matched'})
```

These method calls `create_dictionary` and `create_dictionary_entries` should return the actual instances of `Dictionary` and `DictionaryEntry` but there is no need for the consumer to know how to create these instances. `client` hides that behind a facade and for a good reason.

Reply:
Agreed. I'll rework these parts.